wagtail/admin/templates/wagtailadmin/widgets/tag_widget.html

```diff
@@ -1,4 +1,5 @@
 {% include 'django/forms/widgets/text.html' %}
+<p class="help">{{widget.tag_help_text}}</p>
```

Comment:
```suggestion
<p class="help">{{ widget.help_text }}</p>
```
`help_text` is probably a simpler name, we know it is the tag as this is the tag widget template, also it is good practice for readability to put spaces between the template tagging braces.